nilearn/datasets/struct.py

```diff
@@ -106,38 +114,134 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
-def load_mni152_template():
-    """Load skullstripped 2mm version of the MNI152 originally distributed
-    with FSL.
+def load_mni152_template(resolution=2):
+    """This function takes the skullstripped 1mm-resolution version of the
+    MNI152 T1 template, originally distributed with FSL, and re-samples it
+    using a different resolution, if specified.
 
     For more information, see :footcite:`FONOV2011313`,
     and :footcite:`Fonov2009`.
 
+    Parameters
+    ----------
+    resolution: int, optional, Default = 2
+        If resolution is different from 1, the template is re-sampled with the
+        specified resolution.
+
+        .. versionadded:: 0.8.1
+
     Returns
     -------
-    mni152_template : nibabel object corresponding to the template
+    mni152_template : Nifti1Image, image representing the re-sampled
+        whole-brain template
```

Comment:
We could add a "See also" section below this with references to `load_mni152_gm_template` and `load_mni152_wm_template`, and similarly for these two functions. WDYT?